### ISSUE TITLE
SHOW QUERIES shows NULLed queries from decommissioned nodes

### ISSUE THREAD

**Issue by dendon** [cockroachdb/cockroach#29970]
**BUG REPORT**

Please describe the issue you observed, and any steps we can take to reproduce it:

I just noticed when i ran ```SHOW QUERIES``` that there are NULLed queries from nodes that have been decommissioned.
The nodes are decommissioned for some time now, so I am not sure what caused it, but as far as I can tell, there are no negative impacts from it.

- Which version of CockroachDB are you using?
v2.0.5

```
SHOW QUERIES;
+----------------------------------+---------+----------+----------------------------------+--------------------------------------------------------------------------+----------------------+------------------+-------------+-----------+
|             query_id             | node_id | username |              start               |                                  query                                   |    client_address    | application_name | distributed |   phase   |
+----------------------------------+---------+----------+----------------------------------+--------------------------------------------------------------------------+----------------------+------------------+-------------+-----------+
| NULL                             |       5 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
| NULL                             |       4 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
| NULL                             |       2 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
| NULL                             |       8 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
| NULL                             |      10 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
| NULL                             |      11 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
| NULL                             |       9 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
| NULL                             |       6 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
| NULL                             |       7 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
| NULL                             |       3 | NULL     | NULL                             | NULL                                                                     | NULL                 | NULL             |    NULL     | NULL      |
+----------------------------------+---------+----------+----------------------------------+--------------------------------------------------------------------------+----------------------+------------------+-------------+-----------+
```

**Comment by knz** [comment]
Thanks @dendon for your issue report!

I believe this issue has been addressed in CockroachDB 2.1 via #26821.

Does this answer your request? Let me know if/how we can help you further.

Thank you!

**Comment by dendon** [comment]
Hey @knz, yep, seems to be solved.

Thanks.
